pkg/controller/sub_controller/sub_controller.go

```diff
@@ -125,6 +125,9 @@ func (d *SubDefaultController) CheckConfigMountPath(dcr *dorisv1.DorisCluster, c
 	var mountsMap = make(map[string]dorisv1.MountConfigMapInfo)
 	for _, cm := range cms {
 		path := cm.MountPath
+		if cm.MountPath == "" {
```

Comment:
Empty strings are also expected behavior and should not be treated specially.

Reply:
Ok, I'll modify this.